update-since-todo.sh

```diff
@@ -7,13 +7,11 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-me="$( basename "$0" )"
-
 # Needs bash 4+
 declare -A commitsAndTags
 
 IFS=$'\n'
-for todo in $( git grep --line-number '@since TODO' | grep -v "$me" )
+for todo in $( git grep --line-number '@since TODO' -- *.java )
```

Comment:
FTR jelly sometimes has 'since' in it

Reply:
The cases that I found of Jelly files having `@since` all seemed to be unrelated to Jenkins release versions with only one exception, `war/src/main/webapp/scripts/sortable.js`.  That single exception didn't seem significant enough to justify an entry in the script, though I'm certainly willing to expand the match from `*.java` to `*.java *.jelly`.